frontend/src/types/plan.yaml

```diff
@@ -118,16 +117,13 @@ planList:
   - type: 1
     title: team
     trialDays: 14
-    unitPrice: 0
```

Comment:
Same here, and `unitPrice` is set to 0 because we will enforce subscriptions have al least 1 instance, and cal the price by {instance count} * instance unit price * month